packages/sdk/src/protocols/filter.ts

```diff
@@ -312,6 +403,16 @@ export class SubscriptionManager implements ISubscriptionSDK {
     clearInterval(this.keepAliveTimer);
     this.keepAliveTimer = null;
   }
+
+  private incrementMissedMessageCount(peerIdStr: string): void {
```

Comment:
nit: I find it useful to implement some util data structure like [here](https://github.com/waku-org/js-waku/blob/00635b7afe60c2ed739f2ccd1f07b2a6cc04f797/packages/sdk/src/protocols/base_protocol.ts#L236), this way main class is not getting bloated